packages/js-client/README.md

```diff
@@ -0,0 +1,103 @@
+# TSDX User Guide
```

Comment:
Shouldn't the readme explain how to use the existing client and how to adapt things? Or is it intended for later?

Reply:
Same as before, first I wsaant to end the code then write the readme